docs/CHILI-GraFx/users/roles/index.md

```diff
@@ -1,5 +1,51 @@
 # User management
 
+## Overview of Role Access
+
+We are enhancing our SaaS platform's user management through a phased approach. 
+
+Currently, Subscription Admins and Environment Admins can create templates to familiarize themselves with the system. 
```

Comment:
also TD & CA can do this (see table), so might be confusing to state it like this